docs/src/pages/components/data-grid/rows/rows.md

```diff
@@ -71,6 +71,14 @@ To change the row height for the whole grid, set the `rowHeight` prop:
 
 {{"demo": "pages/components/data-grid/rows/DenseHeightGrid.js", "bg": "inline"}}
 
+## Variable row height
+
+If you need different rows to have different row heights this can be achieved using the `getRowHeight` prop. The prop is called for each visible row and if the return value is `number` then that `number` will be set as that row's `rowHeight`. If the return value is `null` or `undefined` then the `rowHeight` prop will take effect for the given row.
+
+{{"demo": "pages/components/data-grid/rows/VariableRowHeightGrid.js", "bg": "inline"}}
```

Comment:
This demo could have the toolbar so users can play with the density. Also, the preview doesn't show how to calculate the value based on the factor.